docs/demos/dev-bsdate/desc.md

```diff
@@ -0,0 +1 @@
+dev
```

Comment:
I did what all the other "dev-" files did, which is just but "dev" in it.  This is a dev only test and only shows up on the dev.html file.